src/model/ManifestV2.ts

```diff
@@ -108,7 +107,7 @@ export default class ManifestV2 implements ReactiveObjectConverterInterface {
         return this;
     }
 
-    public fromReactive(reactive: any): ManifestV2 {
+    public fromJsObject(reactive: any): ManifestV2 {
```

Comment:
NGL I'm a bit irked about the argument name still being "reactive". Not a blocker but wink wink nudge nudge.